src/rules/ec2/EC2EBSVolumeEncrypted.ts

```diff
@@ -20,34 +23,11 @@ export default Object.defineProperty(
         return NagRuleCompliance.NON_COMPLIANT;
       }
       return NagRuleCompliance.COMPLIANT;
-    } else if (node instanceof CfnLaunchTemplate) {
-      const launchTemplateData = Stack.of(node).resolve(
-        node.launchTemplateData
-      );
-      if (launchTemplateData.blockDeviceMappings == undefined) {
-        return NagRuleCompliance.NON_COMPLIANT;
-      } else {
-        const blockDeviceMappings = Stack.of(node).resolve(
-          launchTemplateData.blockDeviceMappings
-        );
-        for (const blockDeviceMapping of blockDeviceMappings) {
-          const encryption = NagRules.resolveIfPrimitive(
-            node,
-            blockDeviceMapping.ebs.encrypted
-          );
-          if (encryption !== true) {
-            return NagRuleCompliance.NON_COMPLIANT;
-          }
-        }
-        return NagRuleCompliance.COMPLIANT;
-      }
     } else if (node instanceof CfnInstance) {
       const blockDeviceMappings = Stack.of(node).resolve(
         node.blockDeviceMappings
       );
-      if (blockDeviceMappings == undefined) {
-        return NagRuleCompliance.NON_COMPLIANT;
-      } else {
+      if (blockDeviceMappings !== undefined) {
```

Comment:
I believe we still need to return `NON_COMPLIANT` if the blockDeviceMappings are `undefined` and a `LuanchTemplate` is not specified.

If I'm understanding the [CloudFormation documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-instance.html#cfn-ec2-instance-blockdevicemappings) correctly, the root EBS volume may not be encrypted otherwise